repl/src/test/scala/org/apache/spark/repl/SparkShellSuite.scala

```diff
@@ -26,6 +26,8 @@ import scala.concurrent.duration._
 
 import org.apache.spark.ProcessTestUtils.ProcessOutputCapturer
 import org.apache.spark.SparkFunSuite
+import org.apache.spark.internal.{MDC, MessageWithContext}
```

Comment:
We don't need to migrate the log entries from tests.  Let's revert the changes in this file to avoid introducing new log keys.